<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Final Fantasy XII - The Zodiac Age: Weapon Loadout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="characterGpBox">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Character Classes</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="class1GpBox">
       <property name="title">
        <string>Class 1</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="shikariRadBtn1">
          <property name="text">
           <string>Shikari</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="bushiRadBtn1">
          <property name="text">
           <string>Bushi</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="uhlanRadBtn1">
          <property name="text">
           <string>Uhlan</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="knightRadBtn1">
          <property name="text">
           <string>Knight</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="monkRadBtn1">
          <property name="text">
           <string>Monk</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="foeBreakerRadBtn1">
          <property name="text">
           <string>Foe Breaker</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QRadioButton" name="archerRadBtn1">
          <property name="text">
           <string>Archer</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="machinistRadBtn1">
          <property name="text">
           <string>Machinist</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QRadioButton" name="wMageRadBtn1">
          <property name="text">
           <string>White Mage</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="bMageRadBtn1">
          <property name="text">
           <string>Black Mage</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QRadioButton" name="rMageRadBtn1">
          <property name="text">
           <string>Red Mage</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QRadioButton" name="tMageRadBtn1">
          <property name="text">
           <string>Time Mage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="class2GpBox">
       <property name="title">
        <string>Class 2</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="shikariRadBtn2">
          <property name="text">
           <string>Shikari</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="bushiRadBtn2">
          <property name="text">
           <string>Bushi</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="uhlanRadBtn2">
          <property name="text">
           <string>Uhlan</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="knightRadBtn2">
          <property name="text">
           <string>Knight</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="monkRadBtn2">
          <property name="text">
           <string>Monk</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="foeBreakerRadBtn2">
          <property name="text">
           <string>Foe Breaker</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QRadioButton" name="archerRadBtn2">
          <property name="text">
           <string>Archer</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="machinistRadBtn2">
          <property name="text">
           <string>Machinist</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QRadioButton" name="wMageRadBtn2">
          <property name="text">
           <string>White Mage</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="bMageRadBtn2">
          <property name="text">
           <string>Black Mage</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QRadioButton" name="rMageRadBtn2">
          <property name="text">
           <string>Red Mage</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QRadioButton" name="tMageRadBtn2">
          <property name="text">
           <string>Time Mage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="summaryGpBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>282</width>
      <height>423</height>
     </rect>
    </property>
    <property name="title">
     <string>Weapon Summary</string>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="elementsLbl">
       <property name="text">
        <string>Elements Usable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="statusesLbl">
       <property name="text">
        <string>Statuses Applicable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="elementsTxtBrowser"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="statusesTxtBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>402</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Stats</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
